.github/workflows/submodule-pin-check.yml

```diff
@@ -27,8 +27,7 @@ jobs:
           declare -Ar exceptions=(
             [contracts]=origin/v2-main
             [nitro-testnode]=origin/master
-
-            #TODO Rachel to check these are the intended branches.
+            [bold]=main
```

Comment:
Why not "origin/main" like the others?

Reply:
Thanks for catching this, opened https://github.com/OffchainLabs/nitro/pull/2957